planning/behavior_velocity_planner/config/detection_area.param.yaml

```diff
@@ -7,3 +7,4 @@
       use_pass_judge_line: false
       state_clear_time: 2.0
       hold_stop_margin_distance: 0.0
+      distance_to_judge_over_stop_line: 0.5
```

Comment:
@mkuri 

Out of curiosity, why is this value set to 0.5? 

And In my understanding, there are some parameters for stopping with a little margin before the stop line, and tuning those values so as not to cross the stop line is required. With this proposal value, is the idea to tune it so that detection area can be checked even if the vehicle crosses the stop line instead of stopping before the stop line?

If my understanding above is correct, I'm wondering about the usage of tuning to stop before the stop line versus tuning to allow passing over the stop line.

Reply:
@kyoichi-sugahara 
Tuning to stop before the stop line is necessary.
The above parameter is a distance at which it is safer to proceed than to stop because the stop line has been completely crossed.